liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -409,6 +409,7 @@ public Integer run() throws Exception {
                                 }
                             }
                         });
+                        Scope.getCurrentScope().getMdcManager().clear();
```

Comment:
@StevenMassaro do you think I need to do this?

Reply:
Is this to remove the keys you're adding in Liquibase.java?

Why did you move this from line 461?

Reply:
Resolved via chat this line is necessary - best practice is to clean up threadlocal variables so in the event that someone is reusing threads with Liquibase they don't encounter any unexpected behavior. The finally block was removed because it cleans up all custom mdc key value pairs prior to logging the exception.